frontend/src/pages/modelServing/screens/metrics/BiasTab.tsx

```diff
@@ -1,38 +1,90 @@
 import React from 'react';
-import { PageSection, Stack, StackItem } from '@patternfly/react-core';
+import {
+  Bullseye,
+  PageSection,
+  Spinner,
+  Stack,
+  StackItem,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { useParams } from 'react-router-dom';
+import MetricsPageToolbar from '~/pages/modelServing/screens/metrics/MetricsPageToolbar';
+import BiasMetricConfigSelector from '~/pages/modelServing/screens/metrics/BiasMetricConfigSelector';
+import { BiasMetricConfig } from '~/concepts/explainability/types';
 import { useExplainabilityModelData } from '~/concepts/explainability/useExplainabilityModelData';
-import DIRGraph from './DIRChart';
-import MetricsPageToolbar from './MetricsPageToolbar';
-import SPDChart from './SPDChart';
-import EmptyBiasConfigurationCard from './EmptyBiasConfigurationCard';
+import TrustyChart from '~/pages/modelServing/screens/metrics/TrustyChart';
+import BiasMetricChartWrapper from '~/pages/modelServing/screens/metrics/BiasMetricChartWrapper';
+import { useBrowserStorage } from '~/components/browserStorage';
+import EmptyBiasConfigurationCard from '~/pages/modelServing/screens/metrics/EmptyBiasConfigurationCard';
+import EmptyBiasChartSelectionCard from '~/pages/modelServing/screens/metrics/EmptyBiasChartSelectionCard';
+
+const SELECTED_CHARTS_STORAGE_KEY_PREFIX = 'odh.dashboard.xai.selected_bias_charts';
+const OPEN_WRAPPER_STORAGE_KEY_PREFIX = `odh.dashboard.xai.bias_metric_chart_wrapper_open`;
+const BiasTab: React.FC = () => {
+  const { inferenceService } = useParams();
+
+  const { biasMetricConfigs, loaded } = useExplainabilityModelData();
+
+  const [selectedBiasConfigs, setSelectedBiasConfigs] = useBrowserStorage<BiasMetricConfig[]>(
+    `${SELECTED_CHARTS_STORAGE_KEY_PREFIX}-${inferenceService}`,
+    [],
+    true,
+    true,
```

Comment:
Any reason we are going to make this a session storage but not local storage?

Reply:
Vince asked for it to persist for the length of the session. I personally agree with this take, I think if we want to persist the graph state permanently for the user, that should be handled with a backend data store. That means any stale data would get cleaned up for us automatically and also the charts state persists across machines etc. I think session is the right balance for this, the user doesn't lose their state when they refresh, or navigate to a different inference model, which is a real nuisance.